client/src/rpc_client.rs

```diff
@@ -353,6 +353,38 @@ impl RpcClient {
         self.send(RpcRequest::GetVoteAccounts, json!([commitment_config]))
     }
 
+    pub fn wait_for_max_stake(
+        &self,
+        commitment: CommitmentConfig,
+        max_stake_percent: f32,
+    ) -> ClientResult<()> {
+        let mut current_percent;
+        loop {
```

Comment:
Should we implement this loop in CLI?  I can probably consume all of the RPC workers with it here

Reply:
client-side workers?

Reply:
Ah it's just a helper. N/m all good!